src/components/detail/ContentData.vue

```diff
@@ -162,6 +162,7 @@ body[theme-dark] {
 
     table {
       border-color: grey;
+      color: var(--text-dark);
```

Comment:
IMO, instead of adding it here, add a table tag in 
Line#159
```css
h1, h2, h3, h4, h5, h6, p, ul, li, table {
      color: var(--text-dark);
    }
```